### ISSUE TITLE
Align with WebVR where/if it makes sense

### ISSUE THREAD

**Issue by anssiko** [w3c/mediacapture-depth#79]
> The `VREyeParameters` interface represents all the information required to correctly render a scene for a given eye.

http://mozvr.github.io/webvr-spec/webvr.html#vreyeparameters

> The `VRFieldOfView` interface represents a field of view, as given by 4 degrees describing the view from a center point.

http://mozvr.github.io/webvr-spec/webvr.html#vrfieldofview0

We should see if it makes sense to align the naming and/or the API shape:

http://w3c.github.io/mediacapture-depth/#mediatracksettings-dictionary

**Comment by robman** [comment]
While they're conceptually related they're really different descriptions.

The model we've used in mediacapture-depth is aligned with common use in 
Computer Vision and it's goal is to allow us to describe the sensor's 
field of view in the real space in front of it.

The model used in the WebVR spec is more about how the user's eyes 
interact with the displays from the virtual camera/s to generate their 
rendered/visible field of view.

So really we have at least 4 distinct points of view that we're dealing 
with:
- RGB sensor/s
- Depth sensor/s
- Virtual camera/s
- Eye/s

But WebVR only deal with the last 2 (at the moment - but many of the 
HMD's are now adding depth sensors so this may change leading them more 
into our space).

The 4 parameter model used in WebVR also allows for display fov's that 
are not symmetrical about a centre point. In Computer Vision this is 
normally described using Camera Extrinsics. But to simplify the 
mediacapture-depth api we've assumed the centre of the image plane is 
the centre of the sensor's fov (happy to revisit this discussion if we 
think extrinsics are needed too).

 From my perspective, the WebVR model doesn't really map well to 
Computer Vision (e.g. the pinhole camera model). If the 4 params provide 
an asymmetrical viewport then really they're just clipping in one or two 
directions. This is different from how we would describe this in a 
pinhole camera model.

Also, they just seems redundant to me. The example provided[1] ends up 
combining the two matching param pairs (e.g. up/down and left/right) to 
create xscale and yscale params, etc. in order to create a "WebGL 
compatible projection matrix". In fact from reviewing the spec and 
searching the web I can't find any good examples where the independent 4 
params are actually used separately. So this is really just a waste of 
compute cycles.

The WebVR people also have no concept of or need for additional params 
we require like focalLength.

However, it would be good if we could have a general discussion with 
Vlad and Brandon to clarify how the mapping between the two spaces migt 
work and perhaps we could provide some examples to help devs understand 
that more clearly.

Also, @anssik I just noticed that you dropped measureType even after 
@ningxin and I stated that it was needed 8( I'll open an issue on this 
in a sec.

[1] http://mozvr.github.io/webvr-spec/webvr.html#example-b45e6820
